### DISCUSSION TITLE
Show an example of a more advanced query

### DISCUSSION THREAD

**Discussion by henryiii** [PyGithub/PyGithub#1780] [Ideas]
It would have helped me to see something to guide a query like this:

```python
gh.search_issues(
         "is:pull-request is:merged",
         author = user,
         created = f">{start_date}",
    )
```

I guessed the meaning of the `**` praams from the API docs, but had to look into the source to find out what to write instead of `"is:pull-request+is:merged"` (since `is` is not a valid keyword and you can't have two of them anyway) - I found the `" ".join(...)` so I was able to get it, but a little help from the docs here would have been nice. :)

**Comment by s-t-e-v-e-n-k** [comment]
We would welcome any contribution that showed more examples.
